vagrant/provisioning/roles/backup_data/tasks/main.yml

```diff
@@ -0,0 +1,75 @@
+- name: ensure Solr is stopped
+  become: yes
+  systemd:
+    name: solr
+    state: stopped
+
+- name: ensure Alfresco is stopped
+  become: yes
+  systemd:
+    name: alfresco
+    state: stopped
+
+- name: ensure Snowbound is stopped
+  become: yes
+  systemd:
+    name: snowbound
+    state: stopped    
+    
+- name: backup data folders 
+  become: yes
+  file:
+    path: "{{ root_folder }}/backups/data"
+    state: present
+
+- name: Solr data backuping
```

Comment:
I don't think "backuping" is a word in English... Maybe "Backing up Solr data" would be more appropriate?
Ditto for the other ones